site/js/components/ee.js

```diff
@@ -0,0 +1,227 @@
+        hint: 'Check the homepage.',
+        responseCorrect: 'Nice!',
+        responseIncorrect: 'Not quite.',
+      },
+      {
+        question: "What's the codename for Linux on Chrome OS?",
+        hint: 'We mention it on the Linux page.',
+        responseCorrect: 'Nice!',
+        responseIncorrect: 'Hmm, try again.',
+      },
+      {
+        question: 'How many weeks are there between Chrome OS releases?',
+        hint: 'Refer to Chromium Dash.',
+        responseCorrect: 'Nice!',
+        responseIncorrect: "You're almost there!",
+      },
+    ];
+    this.text.answers = ['2011', '50', 'Crostini', '4'];
+    this.text.reward = {
+      headline: 'Huzzah!',
+      body: ["You're officially a Chrome OS insider. We've got more exciting things coming your way for our 100th software release. To stay in the know, sign up for our newsletter, and keep exploring the all-new ChromeOS.dev to help you build your next big idea.", "Thank you for all of your support, we couldn't have reached this milestone without you!", 'The Chrome OS team'],
+    };
+
+    if (!this.quizStart) {
+      console.warn('%c%s', 'font-weight: bold;', 'm100.start()');
+    } else {
+      this._promptQuestion(this.curPrompt);
+    }
+  }
+
+  /**
+   * Get started
+   * @return {void}
+   */
+  start() {
+    if (!this.quizStart) {
+      this._setQuizStart(true);
+      console.info('%c%s', 'font-weight: bold; background: #174ea6; color: white; padding: .25em;', this.text.intro.headline);
+      for (const iterator of this.text.intro.body) {
+        console.info(iterator);
+      }
+    }
+    console.warn('Hint: to answer a question use: %c%s', 'font-family: monospace; font-weight: bold;', 'm100.answer(value);');
+    this._promptQuestion(this.curPrompt);
+  }
+
+  /**
+   * Answer
+   * @param {string} value - Submit an answer to a prompted question
+   * @return {void}
+   */
+  answer(value) {
+    if (!this.quizStart) {
+      console.error('Hmmm... what are you trying to answer?');
+    } else {
+      this._checkAnswer(this.curPrompt, value);
+    }
+  }
+
+  /**
+   * Start over
+   * @return {void}
+   */
+  clear() {
+    this._clearQuizStart();
+    this._clearCurrentPrompt();
+  }
+
+  /**
+   * Gets the current prompt value from local storage.
+   * @return {int} - current prompt value
+   */
+  _getCurrentPrompt() {
+    const curPrompt = parseInt(localStorage.getItem('chromeos-m100-current-prompt'));
+    return curPrompt;
+  }
+
+  /**
+   * Sets the current prompt value.
+   * @param {int} value - the current prompt integer value
+   * @return {void}
+   */
+  _setCurrentPrompt(value) {
+    localStorage.setItem('chromeos-m100-current-prompt', value);
+    this.curPrompt = value;
+  }
+
+  /**
+   * Clear the current prompt value.
+   * @return {void}
+   */
+  _clearCurrentPrompt() {
+    localStorage.removeItem('chromeos-m100-current-prompt');
+    this.curPrompt = 0;
+  }
```

Comment:
Why not remove this in favor of setting the current prompt to 0?

Reply:
Same here I wanted to isolate these methods and allow for future extensibility in case we need to clear other things tied to current step (something we haven't thought of yet?)